test/cmd/legacy-script.sh

```diff
@@ -885,6 +885,11 @@ runTests() {
     record_command run_kubectl_explain_tests
   fi
 
+  ###########################
+  # CRD Deletion / Re-creation #
+  ###########################
+
+  record_command run_crd_deletion_recreation_tests
```

Comment:
Hmm, another comment...

Looking at how the other tests are called, I think you probably should wrap this in a check, like the others do, to check if namespaces are supported (?).  For example:

```
  if kube::test::if_supports_resource "${namespaces}" ; then
    record_command run_crd_deletion_recreation_tests
  fi
```

I say this because your tests create a namespaced resource, so I guess it makes sense to check this.  However, I'm not entirely sure when that condition would ever be false.